website/install.sh

```diff
@@ -153,7 +153,8 @@ if [ ! -d startrails/thumbnails -o ! -d keograms/thumbnails -o ! -d videos/thumb
 	echo
 
 	echo -e "${GREEN}* Fixing ownership and permissions${NC}"
-	chown -R pi:www-data .
+	U=$(id --name --user)
```

Comment:
Can we use `U=$(id -nu)`? 
The reason I ask is because I just tried in the Mac terminal and `--name` and `--user` are not valid options. I'm trying to keep things as portable as possible. At some point you could run allsky on Ubuntu. Not sure if that's still the case though.

Reply:
Just for you Thomas.  I changed to "-n -u" since to me that's clearer.